commerce-add-ons/payment-providers/quickpay/release-notes.md

```diff
@@ -1,11 +1,11 @@
 # Release Notes
 
-In this section, we have summarized the changes to Quickpay Payment Provider for Commerce released in each version. Each version is presented with a link to the [Commerce Quickpay Payment Provider issue tracker](https://github.com/umbraco/Umbraco.Commerce.PaymentProviders.Quickpay/issues) showing a list of issues resolved in the release.  We also link to the individual issues themselves from the detail.
+In this section, we have summarized the changes to Quickpay Payment Provider for Umbraco Commerce released in each version. Each version is presented with a link to the [Commerce Quickpay Payment Provider issue tracker](https://github.com/umbraco/Umbraco.Commerce.PaymentProviders.Quickpay/issues) showing a list of issues resolved in the release.  We also link to the individual issues themselves from the detail.
 
 If there are any breaking changes or other issues to be aware of when upgrading they are also noted here.
 
 {% hint style="info" %}
-For details of releases for **Quickpay Payment Provider for Vendr**, refer to the [Change log file on Github](../../changelog-archive/quickpay.md).
+For details of changes for **Quickpay Payment Provider for Umbraco Commerce**, refer to the [Releases section on Github](https://github.com/umbraco/Umbraco.Commerce.PaymentProviders.Quickpay/releases).
```

Comment:
there might be some people still using Vendr and this is the only way they can find the release notes for Vendr. I have added below the main link
```suggestion
For details of releases for **Quickpay Payment Provider for Vendr**, refer to the [Change log file on Github](../../changelog-archive/quickpay.md).
```